.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to pyccel
+
+We welcome any and all contributions to pyccel!
+
+The following is a summary of the best ways to interact/help with this project.
+
```

Comment:
great work @EmilyBourne.
What do you think about restructuring the introduction to contain the following:
- Appreciating the desire for contributing to the pyccel project.
- Giving some resources to start with to know more about the project before starting.
- then proceeding with the ways of contributing to Pyccel.

Reply:
> Giving some resources to start with to know more about the project before starting.

What resources did you have in mind?

Reply:
Hmm, I think most of those are about contributing code. That might be intimidating for someone who just wants to make sure their issue looks right.

I think these links should go in the developer docs

Reply:
Or it's a duplicate of the README

Reply:
hmm, yes I agree with you, let's keep it like that.